live-examples/html-examples/text-content/p.html

```diff
@@ -0,0 +1,8 @@
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Maecenas viverra facilisis rutrum</p>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. <br>
```

Comment:
Nit: Code style: This is not documented yet but, I prefer that we explicitly close self closing tags. Even though it is not required in HTML5. Thanks